30413/Mihai Alexandru Sandu/Lab07/README.md

```diff
@@ -1,5 +1,8 @@
-# Lab X
+# Lab 07
 
-> Edit this file and describe in a few words what you've done in this lab.
+> In this lab we talked about graphs, learned how to implement them in 3 different ways
+> Also learned how 2 traversals work, BFS and DFS
+> learned how to implement Dijkstra’s algorithm
+>P.S i will send the extra assigment after the deadline with like 20 minutes, I hope it still counts :(
```

Comment:
It counts.

Please briefly explain the used strategy for solving the extra problem.